docs/reference.md

```diff
@@ -605,6 +671,7 @@ flow.
 * `JCB` - Japan Credit Bureau credit card.
 * `CHINA_UNIONPAY` - China UnionPay credit card.
 * `SQUARE_GIFT_CARD` - [Square-issued gift card].
+* `EFTPOS` - EFTPos.
```

Comment:
What does "EFTPos" mean? This description doesn't actually tell me anything

Reply:
I'm using same description:
https://docs.connect.squareup.com/api/reader/ios/Enums/SQRDCardBrand.html#/c:@E@SQRDCardBrand@SQRDCardBrandEftpos

Reply:
We shouldn't replicate bad behavior just because we failed to document it well in the in the tech ref. I'm assuming this is related to a debit transaction or cash-back request (we should confirm with the mobile team that this is accurate). If so, I would suggest:

A debit or cash-back transaction (Electronic Funds Transfer at Point Of Sale).